src/fetch_business_partners_info.py

```diff
@@ -0,0 +1,104 @@
+import os
+import json
+import datetime
+import time
+import pandas as pd
+from unicodedata import normalize
+from selenium import webdriver
+from bs4 import BeautifulSoup
+
+
+def selenium_driver_path():
+    for path in ['chromedriver.exe', 'chromedriver']:
```

Comment:
I'd wouldn't say that users _need_ to be able to choose, but we must make sure that if someone doesn't have Chrome (or maybe Firefox) the script will try to find an alternative (even if the alternatives are restricted to Chrome and Firefox if it's difficult to address Safari, IE, whatever). 

About the Docker error message: I'd add a line to the documentation (of the script itself within `CONTRIBUTING.md`) and also in the `raise` threw by this method here.

Finally, CAPTCHA: I kind of imagined that scenario. But… how many times (from your  experience) one has to enter the CAPTCHA to go through the whole script? And how long dos the full process take?